app/models/hubstats/comment.rb

```diff
@@ -8,7 +8,7 @@ class Comment < ActiveRecord::Base
       .group("hubstats_comments.user_id")
     }
 
-    scope :created_since, lambda {|time| where("created_at > ?", time) }
+    scope :created_in_date_range, lambda {|start_date, end_date| where("hubstats_comments.created_at > ? AND hubstats_comments.created_at < ?", start_date, end_date)}
```

Comment:
:+1: It's faster too if I remember correctly. Good call.

On Mon, Jun 22, 2015 at 1:33 PM, Emma Sax <notifications@github.com> wrote:

> In app/models/hubstats/comment.rb
> <https://github.com/sportngin/hubstats/pull/65#discussion_r32967405>:
>
> > @@ -8,7 +8,7 @@ class Comment < ActiveRecord::Base
> >        .group("hubstats_comments.user_id")
> >      }
> >
> > -    scope :created_since, lambda {|time| where("created_at > ?", time) }
> > +    scope :created_in_date_range, lambda {|start_date, end_date| where("hubstats_comments.created_at > ? AND hubstats_comments.created_at < ?", start_date, end_date)}
>
> What if we used BETWEEN? It'd be shorter, and it's inclusive:
>
> http://stackoverflow.com/questions/749615/does-ms-sql-servers-between-include-the-range-boundaries
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sportngin/hubstats/pull/65/files#r32967405>.
>



-- 
*Andy Fleener*
SOFTWARE ENGINEER
612.237.1030 | LinkedIn
<https://www.linkedin.com/profile/view?id=42332532&trk=nav_responsive_tab_profile>

Sport Ngin | 1400 Van Buren St NE | Minneapolis, MN 55413
sportngin.com | twitter.com/sportngin | facebook.com/sportngin